README.md

```diff
@@ -222,6 +230,16 @@ $ ./test-gui.sh olm
 
 Remove the `--headless` flag to Chrome (chromeOptions) in [protractor.conf.ts](frontend/integration-tests/protractor.conf.ts) to see what the tests are actually doing.
 
+##### Debugging Integration Tests
+
+1. `cd frontend; yarn run build`
+2. Add `debugger;` statements to any e2e test
+3. `yarn run debug-test-suite --suite overview`
+4. Chrome browser URL: 'chrome://inspect/#devices', click on '[inspect](url)' link in **Target (v10...)** section.
```

Comment:
I don't think the markdown for this line renders properly.

Reply:
In what way?  It renders as:

![image](https://user-images.githubusercontent.com/12733153/54927862-62983700-4ee9-11e9-9bf2-a96e38a04219.png)

Is the 'inspect' link throwing you?  I made it a link just so devs would know it's a link (even though 'inspect' link doesn't actually work in the README.

Reply:
Let's not purposely add a broken link to the README :)

Reply:
Ok, removed link